docs/io/output/progress_bars.rst

```diff
@@ -0,0 +1,11 @@
+********************************
+Progress Bars for Simulation Run
+********************************
+
+TARDIS displays progress bars by default to track the simulation's progress.
+
+.. image::
+    ../images/progress_bars_demo.gif
+    :width: 750
+    
+Progress bars can be turned off via the keyword argument ``show_progress_bars=False`` in the ``run_tardis()`` function (see :doc:`../../quickstart`) or in the ``Simulation.from_config()`` class method (see :doc:`../../physics/setup/index`).
```

Comment:
Just one small thing, when https://github.com/tardis-sn/tardis/pull/1823 is merged, this and other text would have to be changed accordingly
```suggestion
Progress bars can be turned on via the keyword argument ``show_progress_bars=True`` in the ``run_tardis()`` function (see :doc:`../../quickstart`) or in the ``Simulation.from_config()`` class method (see :doc:`../../physics/setup/index`).
```